.vscode/settings.json

```
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

[nitpick] I highly recommend deleting this file from your pull request.
```suggestion
    "git.ignoreLimitWarning": true
```

---

Noted with thanks.